cluster_test.go

```diff
@@ -21,6 +21,7 @@ var (
 	testKafkaRoot  = "kafka_2.12-1.0.0"
 	testKafkaAddrs = []string{"127.0.0.1:29092"}
 	testTopics     = []string{"topic-a", "topic-b"}
+	testTopicsReset     = "topic-c"
```

Comment:
pls use `gofmt` or `goimports`

Reply:
also, no real need to declare this as global, it's only used once

Reply:
True. Removed assuming we dont need another fuzzing test.